continuum/datasets/rainbow_mnist.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+
+from continuum.datasets import MNIST
+
+
+class RainbowMNIST(MNIST):
+    def __init__(
+        self,
+        *args,
+        color: str = "red",
```

Comment:
Could we create RainbowMNIST dataset with any color, for example by providing a hex encoding of a color?

Reply:
Not right now, it's a fair remark but I think it should be done in a later PR. I'll open an issue about it